src/Avatar/Avatar.component.tsx

```diff
@@ -70,7 +75,9 @@ export const Avatar = ({
     <StyledAvatar className={classNames('anchor-avatar', className)} {...props}>
         <InnerBorder className="avatar-container">
             {/* TODO: handle image src */}
-            {label ? label.substr(0, 2).toUpperCase() : <DefaultAvatar />}
+            {label && label.substr(0, 2).toUpperCase()}
+            {src && <img src={src} alt="Anchor Avatar" />}
```

Comment:
Could be nice to allow the user to enter in their own `alt` and default to "Anchor Avatar" if they supply nothing. We already let them customize the rest of the content.

Reply:
👍